### DISCUSSION TITLE
Custom wallpapers per-group (as opposed to per-screen)

### DISCUSSION THREAD

**Discussion by kovasap** [qtile/qtile#3515] [Ideas]
Right now in qtile wallpapers can be [specified per screen](https://docs.qtile.org/en/latest/manual/config/screens.html?highlight=wallpaper#screen).  It would be cool if instead they were specified per-group, so that as you cycled through your groups the wallpaper would update accordingly.

**Comment by elParaguayo** [comment]
#2782

**Comment by kovasap** [comment]
Thanks for the link!  I did search for this question before posting I
swear...
